### PR TITLE
Triad bingo

### PR DESCRIPTION
**Pull request by crookedneighbor**
@lemoness @deilann @taldin

I think this is done.

Do we want to clarify in the keys to the kennel text that Triad Bingo only stacks if you have all 90 mounts and all 90 pets twice (IE, no semi-transparent pets)?

Requires: https://github.com/HabitRPG/habitrpg-shared/pull/442

### CODE REVIEW

**Comment by lemoness** [comment]
Yes, I think we should clarify that.

Well done! I'll work on a Bailey :D Ping me when it has been pushed.

File: public/js/controllers/inventoryCtrl.js [hunk 1]
```diff
@@ -91,6 +98,16 @@ habitrpg.controller("InventoryCtrl",
         User.user.achievements.beastMaster = true;
         $rootScope.openModal('achievements/beastMaster');
       }
+
+      // Checks if Triad Bingo has been reached for the first time
+      if(!User.user.achievements.triadBingo
+          && Shared.countPets(null, User.user.items.pets) >= 90
+          && Shared.countMounts(null, User.user.items.mounts) >= 90
+          && $scope.calcTriadBingo()) {
```
Comment (crookedneighbor) [inline public/js/controllers/inventoryCtrl.js:109]: Should `$scope.calcTriadBingo()` just be a function local to $scope.hatch? 

Also, I suppose I could include the `countPets` and `countMounts` checks in it, so the if would just be `(!User.user.achievements.triadBingo && $scope.calcTriadBingo())`. Including them in the function check would also allow it to be used elsewhere, if needed.

**Comment by crookedneighbor** [comment]
How's this (I've bolded the part I changed):

> Miss the thrill of collecting pets? Now you can let them go, and have those drops be meaningful again!
> 
> Use the Key to the Kennels to reset your non-quest collectible pets and/or mounts to zero. (Quest-only and Rare pets and mounts are not affected.)
> 
> There are three Keys to the Kennels: Release Pets Only (4 Gems), Release Mounts Only (4 Gems), or Release Both Pets and Mounts (6 Gems).  Using a Key lets you stack the Beast Master and Mount Master **achievements. The Triad Bingo achievement will only stack if you use the "Release Both Pets and Mounts" key and have collected all 90 pets a second time. Show** the world just how much of collection master you are! But choose wisely, because once you use a Key and open the kennel or stable doors, you won't be able to get them back without collecting them all again...

**Comment by crookedneighbor** [comment]
Ping @lemoness

**Comment by taldin** [comment]
Looks good to me!

**Comment by lemoness** [comment]
Looks good!

**Pull request merged**